### PR TITLE
added about-us page

### PR DESCRIPTION
**Pull request by Gomanius**

### CODE REVIEW

File: src/components/platform/abous-us/AboutUs.tsx [hunk 1]
```diff
@@ -6,7 +6,90 @@ import "./about-us.scss";
 export class AboutUs extends Component<any, any> {
 
     render() {
-        return "Über uns";
+        return (<>
+        <header className="head">
+        <h2>Über Uns</h2><br/>
+
+    </header>
```
Comment (Geist5000) [inline src/components/platform/abous-us/AboutUs.tsx:13]: bad formatting, please fix indentation

File: src/components/platform/abous-us/about-us.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+.table1 td, th {
+    border: 1px solid transparent;
+    text-align: left;
+    
+}
+
+.head h2 {
+      font-size: 40px;
+      color: black;
+}
```
Comment (Geist5000) [inline src/components/platform/abous-us/about-us.scss:15]: add empty line at end of file

**Review by Geist5000 — APPROVED** [review]

File: src/components/platform/abous-us/about-us.scss [hunk 3]
```diff
@@ -0,0 +1,13 @@
+.table1 {
+    width: 20%;
+}
+
+.table1 td, th {
+    border: 1px solid transparent;
```
Comment (nic-schi) [inline src/components/platform/abous-us/about-us.scss:6]: i assume you did transparent here to have a little spacing between the cells. if you want to do that, you could also do "border-spacing: 1px", which would be applied to the table element.

**Review by nic-schi — APPROVED** [review]
Approved

**Pull request merged**